bears/general/QuotesBear.py

```diff
@@ -0,0 +1,66 @@
+from coalib.bears.LocalBear import LocalBear
+from bears.general.AnnotationBear import AnnotationBear
+from coalib.results.Diff import Diff
+from coalib.results.HiddenResult import HiddenResult
+from coalib.results.Result import Result
+
+
+class QuotesBear(LocalBear):
+
+    BEAR_DEPS = {AnnotationBear}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting'}
+
+    def correct_single_line_str(self, filename, file, sourcerange,
+                                preferred_quotation):
+        str_contents = file[sourcerange.start.line - 1][
+                       sourcerange.start.column:sourcerange.end.column-1]
+
+        if preferred_quotation in str_contents:
```

Comment:
If my preferred_quotation is `'` and the tested string is `"Man, I love McDonald's burgers"` this would yield no result although it's incorrectly quoted, right?